pkg/platform/local/platform.go

```diff
@@ -714,13 +648,26 @@ func (p *Platform) GetContainerNameByCreateFunctionOptions(createFunctionOptions
 }
 
 func (p *Platform) getContainerHTTPTriggerPort(container *dockerclient.Container) int {
-	ports := container.HostConfig.PortBindings["8080/tcp"]
-	if len(ports) == 0 {
+	functionHostPort := dockerclient.Port(fmt.Sprintf("%d/tcp", abstract.FunctionContainerHTTPPort))
+
+	portBindings := container.HostConfig.PortBindings[functionHostPort]
+	ports := container.NetworkSettings.Ports[functionHostPort]
+	if len(portBindings) == 0 && len(ports) == 0 {
```

Comment:
did you mean || ?

Reply:
no, one of them can be empty, the other one can be filled